voucher/src/main/java/org/prgms/voucher/AmountType.java

```diff
@@ -0,0 +1,14 @@
+package org.prgms.voucher;
+
+public enum AmountType {
+    FIX("fix", 1),
+    PERCENT("percent", 2);
+
+    private final String name;
+    private final int number;
```

Comment:
```suggestion
    private final String voucherType;
    private final int consoleChoiceNumber;
```
좀 더 명시적인 이름을 사용하면 좋겠다는 생각이 들어요!

Reply:
넵!! 확실히 의미가 명확하게 표현되는 것 같습니다😄 수정하겠습니다!!

Reply:
다른 종류의 상품권으로 사용되거나 애플리케이션으로 사용될 가능성을 고려해서 `amountType`, `choiceNumber`로 수정하였습니다!